shell/common/api/electron_api_native_image_mac.mm

```diff
@@ -103,12 +105,102 @@ void ReceivedThumbnailResult(CGSize size,
   return handle;
 }
 
+gin_helper::Handle<NativeImage> NativeImage::CreateMenuSymbol(
+    gin::Arguments* args,
+    std::string name) {
+  @autoreleasepool {
+    float pointSize = 13.0;
+    NSFontWeight weight = NSFontWeightSemibold;
+    NSImageSymbolScale scale = NSImageSymbolScaleSmall;
+
+    NSImage* image =
+        [NSImage imageWithSystemSymbolName:base::SysUTF8ToNSString(name)
+                  accessibilityDescription:nil];
+
+    NSImageSymbolConfiguration* symbol_config =
+        [NSImageSymbolConfiguration configurationWithPointSize:pointSize
+                                                        weight:weight
+                                                         scale:scale];
+
+    image = [image imageWithSymbolConfiguration:symbol_config];
+
+    NSData* png_data = bufferFromNSImage(image);
+
+    gin_helper::Handle<NativeImage> handle =
+        CreateFromPNG(args->isolate(), electron::util::as_byte_span(png_data));
+
+    gfx::Size size = handle->GetSize(1.0);
+
+    float aspect_ratio =
+        static_cast<float>(size.width()) / static_cast<float>(size.height());
+
+    int new_height = 14;
+    int new_width = static_cast<int>(14 * aspect_ratio);
+
+    gin_helper::Handle<NativeImage> sized = handle->Resize(
+        args,
+        base::Value::Dict().Set("width", new_width).Set("height", new_height));
```

Comment:
Is this resize necessary if you already set the point size to 13?

Reply:
Yes, without the resize the image tries to render itself at 2x size.